client/app/components/calendar/event/DeleteConfirmation.jsx

```diff
@@ -1,17 +1,45 @@
 import React from 'react';
 import { Modal, Button } from 'react-bootstrap';
+import EventSchema from 'schemas/EventSchema';
+import { eventTimeString } from 'helpers/DateHelper';
+import moment from 'moment';
 
-const DeleteConfirmation = ({ onHide, show, cancelHandler, confirmHandler }) => (
-  <Modal bsSize="small"
-         show={show}
+const TIME_FORMAT = 'HH:mm';
+const DATE_FORMAT = 'Mo MMM YYYY';
+
+const DeleteConfirmation = ({ onHide, show, cancelHandler, confirmHandler, event }) => (
```

Comment:
why not `onCancel` and `onCofirm` instead of `cancelHandler` and `confirmHandler`?